elixirkit/lib/elixirkit/bundler.ex

```diff
@@ -0,0 +1,255 @@
+defmodule ElixirKit.Bundler do
+  @moduledoc false
+
+  @openssl_version "1.1.1s"
+  @otp_version "25.1.2"
+  @root "#{__DIR__}/../.."
+  @tmp "/tmp/elixirkit"
```

Comment:
```suggestion
  @tmp Path.join(System.tmp_dir!(), "elixirkit")
```

Reply:
Good call. This reminds me, this shouldn't be a tmp dir but a cache dir and a different path. Bootstrapping OTP is pretty slow. We definitely want it to persist across system restarts!